client/pages/product/addproduct.jsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState, useEffect } from 'react';
+import { axiosInstance } from '../../configs/axios';
+
+const CreateProduct = () => {
+  const [product, setProduct] = useState({
+    name: '',
+    description: '',
+    image: '',
+    price: '',
+    intervals: 'monthly', // Default interval
+  });
+
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+
+    try {
+      const token = window.localStorage.getItem('token');
+
+      const { name, description, image, price, intervals} = product;
+
+      const responseData = await axiosInstance({
```

Comment:
I remember suggesting axios interceptors to fetch and attach `Authorization` header. Do not do it here in the component. 

[https://axios-http.com/docs/interceptors](https://axios-http.com/docs/interceptors)